spec/special-vars.test.sh

```diff
@@ -225,6 +225,17 @@ f
 ['4']
 ## END
 
+#### $LINENO and redirections (bug regression)
+>_tmp/lineno_regression$LINENO
+ls _tmp/lineno_regression1
```

Comment:
The spec test frame work sets `$TMP` and that's preferrred, because some tests in `test/spec.sh` pass `--cd-tmp`, which makes the test more isolated from the file system.

Reply:
Gotcha.  But I do see some tests that have hardcoded references to _tmp/ -- are those just older and haven't been updated?